libqtile/resources/default_config.py

```diff
@@ -122,6 +123,9 @@
 )
 extension_defaults = widget_defaults.copy()
 
+wallpaper_path = Path(__file__).resolve() / ".." / "qtile-wallpaper.png"
```

Comment:
Maybe stick it in resources/ alongside the default config?

Reply:
That's exactly where it is.

Reply:
Oh yeah, brainfart :D